client/src/components/History/Content/ContentItem.vue

```diff
@@ -0,0 +1,202 @@
+<template>
+    <div
+        :id="contentId"
+        :class="['content-item m-1 p-0 rounded', clsStatus]"
+        draggable
+        :data-hid="id"
+        :data-state="state"
+        @dragstart="onDragStart">
+        <div class="p-1 cursor-pointer" @click.stop="onClick">
+            <div class="clearfix overflow-hidden">
+                <div class="btn-group float-right">
+                    <b-button
+                        v-if="expandable && state !== 'noPermission' && state !== 'discarded'"
```

Comment:
I was wondering the same but these conditions and states are as declared in the previous code (see: https://github.com/galaxyproject/galaxy/blob/dev/client/src/components/History/model/states.js#L27). I kept them in order to avoid regression.

Reply:
Renamed `expandable` to `isDataset`, and `expanded` to `expandDataset`.

Reply:
Revised state handling. Please see: https://github.com/galaxyproject/galaxy/pull/13477/commits/e72efd1ee09e512be180d3f68880ccf2edeaa937.